### DISCUSSION TITLE
Feature Request: Automatically add show year to scene/show exception list

### DISCUSSION THREAD

**Discussion by flick1999** [pymedusa/Medusa#11631] [Ideas & Feature requests]
I'm noticing that more and more of the scene releases are adding the show year to the file name, even if the show's name is unique. I've added several of them manually as Show Exceptions in the Scene Exception field, then realized it might not be a terrible idea for Medusa to just do that automatically, either visually on that page or behind the scenes.

Thank you.

**Comment by nydave69** [comment]
I just ran into the issue of 'The Diplomat' a currently running US show.
It matches other show with similar names. Medusa likes to jumble the episodes all around.
Guess it doesn't help..      Added a ticket for my issue.
As with others I have come across,

**Comment by flick1999** [comment]
Seeing more and more shows show up with the year in the filename. Just bumping this accordingly. Thank you.

**Comment by nydave69** [comment]
I use the following format for the show;
Worlds Apart (US) (2003) [eng] {imdb-tt0381832} {tmdb-27582} [eng]
 and then the episodes;
Worlds Apart (US) (2003) (S01E02) (2025-10-14) [eng] (720p-AVC BluRay AAC-2ch) Episode 2
